src/Symfony/Component/Process/Tests/ExecutableFinderTest.php

```diff
@@ -159,6 +159,18 @@ public function testEmptyDirInPath()
         unlink('executable');
     }
 
+    public function testFindBuiltInCommandOnWindows()
+    {
+        if ('\\' !== \DIRECTORY_SEPARATOR) {
+            $this->markTestSkipped('Can be only tested on windows');
+        }
+
+        $finder = new ExecutableFinder();
+        $this->assertSame('rmdir', $finder->find('RMDIR'));
```

Comment:
AppVeyor returns `RMDIR` but not `rmdir` (see the failing test). Do we need to fix the expectation here or is there still some flaw in the code?

Reply:
the command names are case insensitive so I guess the implementation is fine but test expectations should be fixed to match the input.

Reply:
👍 thanks for the feedback (see #58748 then)